python/tests/integration/arcticdb/test_arctic.py

```diff
@@ -557,14 +557,32 @@ def test_delete_date_range(arctic_library):
 
 
 @MONGO_TESTS_MARK
-def test_mongo_repr(mongo_test_uri):
-    max_pool_size = 10
-    min_pool_size = 100
-    selection_timeout_ms = 1000
-    uri = f"{mongo_test_uri}/?maxPoolSize={max_pool_size}&minPoolSize={min_pool_size}&serverSelectionTimeoutMS={selection_timeout_ms}"
+def test_mongo_repr_and_uri(mongo_test_uri):
+    uri = f"{mongo_test_uri}/?maxPoolSize=10&minPoolSize=100&serverSelectionTimeoutMS=1000"
     ac = Arctic(uri)
     assert repr(ac) == f"Arctic(config=mongodb(endpoint={mongo_test_uri[len('mongodb://'):]}))"
 
+    # With pymongo, exception thrown in the uri_parser;
+    with pytest.raises(UserInputException):
+        uri = f"{mongo_test_uri}//"
+        ac = Arctic(uri)
+
+
+@MONGO_TESTS_MARK
+def test_mongo_repr_uri_no_pymongo(monkeypatch, mongo_test_uri):
+    import arcticdb.adapters.mongo_library_adapter
+
+    monkeypatch.setattr(arcticdb.adapters.mongo_library_adapter, "_HAVE_PYMONGO", False)
+
+    uri = f"{mongo_test_uri}/?maxPoolSize=10&minPoolSize=100&serverSelectionTimeoutMS=1000"
+    ac = Arctic(uri)
```

Comment:
Makes sense to check we can construct with an OK URI, but we don't need the repr assertion here too

Reply:
Same reason for the above comment. The repr assertion is for making sure the endpoint extraction is correct or not.